docs/dm/zh_CN/features/binlog-filter.md

```diff
@@ -17,9 +26,9 @@ filters:
     ​action: Ignore
 ```
 
-#### 参数解释
+### 参数解释
 
-- [`schema-pattern` / `table-pattern`](./table-selector.md): 对匹配上该规则的上游 MySQL/MariaDB 实例的表的 binlog events 或者 DDL SQLs 进行以下规则过滤
+- [`schema-pattern` / `table-pattern`](./table-selector.md): 对匹配上的上游 MySQL/MariaDB 实例的表的 binlog events 或者 DDL SQLs 进行以下规则过滤
```

Comment:
do we need to add some description about the behavior where more than one rules matched?
some guys have asked me about this.

Reply:
```
- 同一个表匹配上多个规则，将会顺序应用这些规则，并且黑名单的优先级高于白名单，即如果同时存在规则 `Ignore` 和 `Do` 应用在某个 table 上，那么 `Ignore`
```

Reply:
sounds good. you can add this to the doc.